var/spack/repos/builtin/packages/py-dateparser/package.py

```diff
@@ -0,0 +1,33 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyDateparser(PythonPackage):
+    """dateparser -- python parser for human readable dates"""
+
+    homepage = "https://github.com/scrapinghub/dateparser"
+    url      = "https://pypi.io/packages/source/d/dateparser/dateparser-0.7.2.tar.gz"
+
+    version('0.7.2', sha256='e1eac8ef28de69a554d5fcdb60b172d526d61924b1a40afbbb08df459a36006b')
+
+    variant('calendars', default=True, description='Add calendar libraries')
+
+    depends_on('python@2.7:2.8,3.5:', type=('build', 'run'))
+    depends_on('py-setuptools', type='build')
+    depends_on('py-python-dateutil@2.7.5:', type=('build', 'run'))
+    depends_on('py-pytz@2018.9:', type=('build', 'run'))
+    depends_on('py-regex@2019.01.24:', type=('build', 'run'))
+    depends_on('py-tzlocal@1.5.1:', type=('build', 'run'))
+    depends_on('py-umalqurra@0.2:', type=('build', 'run'), when='+calendars')
+    depends_on('py-ruamel-yaml', type=('build', 'run'), when='+calendars')
+    depends_on('py-convertdate@2.1.3:', type=('build', 'run'), when='+calendars')
+    depends_on('py-jdatetime@3.1.0:', type=('build', 'run'), when='+calendars')
```

Comment:
I would drop all of the version requirements as they aren't present in the `setup.py`.

Reply:
done, I think that we will most likely miss some constraints that are actually required, but not enforced. As the typical use case will be "give me the newest" that likely doesn't matter. However should we put that philosophy in the PythonPackage stub?

Reply:
My philosophy is that if older versions of the dependencies don't work, Spack isn't broken, the `setup.py` is. If I found my version of xyz was too old, I would open an issue with the developers and fix the Spack package later.

Reply:
Should I add a couple of comment lines to the PythonPackage stub? Or do you think that is too bloated?

Reply:
Nah, I think the PythonPackage build docs should suffice. I wonder if we should add a link to those in each build system stub...

Reply:
That would be helpful, I think I "learned" spack before the doc was in place and didn't think about going back to the documentation